shell/dialog/AddCustomBadgeDialog.vue

```diff
@@ -312,15 +312,23 @@ export default {
         />
         <div class="buttons">
           <button
+            role="button"
             class="btn role-secondary mr-10"
+            :aria-label="t('generic.cancel')"
             @click="close"
+            @keyup.enter="close"
+            @keyup.space="close"
           >
             {{ t('generic.cancel') }}
           </button>
           <AsyncButton
             :action-label="t('clusterBadge.modal.buttonAction')"
             :disabled="!canSubmit"
+            role="button"
+            :aria-label="t('generic.cancel')"
             @click="apply"
+            @keyup.enter="apply"
+            @keyup.space="apply"
```

Comment:
`AsyncButton` exposes a button at its root, do we need to explicitly specify `enter` and `space` here?

Reply:
true that. Removed